### DISCUSSION TITLE
Kurs aufzeichnen

### DISCUSSION THREAD

**Discussion by acton59** [Courseplay/Courseplay_FS22#2586] [Ideas]
Hallo,
wäre es möglich einen Kurs aufzuzeichnen für die Ballenpresse z.B. den man dann wiese abfahren kann?

acton59

**Comment by Tensuko** [comment]
Kurs aufzeichnen gibt es nicht und wird es nicht geben.
Einen Kurs für den Schwader generieren, abfahren lassen und diesen kopieren um die Ballenpresse den Kurs abfahren zu lassen.
